3-UsingAPIs/Week2/homework/ex1-programmerFun/index.js

```diff
@@ -17,27 +17,40 @@
 ------------------------------------------------------------------------------*/
 function requestData(url) {
   // TODO return a promise using `fetch()`
+  return fetch(url)
+    .then((response) => response.json())
+    .catch((error)   => console.error('Error:', error));
 }
 
 function renderImage(data) {
   // TODO render the image to the DOM
   console.log(data);
+  const renderImage = document.createElement('img');
```

Comment:
Some mini remarks:
- I'd just call the variable `image` or `img` or something, instead of `renderImage`, because the latter sounds like it's a function that _does_ the rendering, instead of being the image itself
- Using `setAttribute` is 👍 -- but you can also use `image.src = data.img` if you want to save some typing :)